src/aspect/event/ExcreteEPS.java

```diff
@@ -42,14 +42,14 @@ public void start(AspectInterface initiator,
 		 * Find out how much EPS the agent can hold before it must excrete.
 		 */
 		double maxEPS = initiator.getDouble(this.MAX_INTERNAL_EPS);
+		
 		/*
 		 * Vary this number randomly by about 10%. If the agent has less EPS
 		 * than this amount, then exit.
 		 */
-		// TODO this should probably be set when the agent has its max EPS		
```

Comment:
this note is not obselete

Reply:
Here the best solution would be to store the agent's EPSblob size after the division and then use that stored value.

Reply:
I've created an issue for this change #217